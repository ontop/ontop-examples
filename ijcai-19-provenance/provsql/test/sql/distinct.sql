\set ECHO none
SET search_path TO public,provsql;

CREATE TABLE distinct_result AS
  SELECT *, formula(provenance(),'personnel_name')
  FROM (
    SELECT DISTINCT classification FROM personnel
  ) t;

SELECT remove_provenance('distinct_result');
SELECT * FROM distinct_result ORDER BY classification;
DROP TABLE distinct_result;
