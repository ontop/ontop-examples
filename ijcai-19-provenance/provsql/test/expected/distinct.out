\set ECHO none
 remove_provenance 
-------------------
 
(1 row)

 classification |     formula     
----------------+-----------------
 unclassified   | John
 restricted     | (Paul ⊕ Nancy)
 confidential   | Dave
 secret         | (Ellen ⊕ Susan)
 top_secret     | Magdalen
(5 rows)

